### ISSUE TITLE
styx QuickStart failing

### ISSUE THREAD

**Issue by ldeck** [styx-static/styx#42]
Styx QuickStart is failing at the following attribute: `server = "${caddy.bin}/bin/caddy”;`.

Should this instead be `server = "${caddy}/bin/caddy”;`?

```
$ nix-shell -p $(nix-build https://github.com/styx-static/styx/archive/latest.tar.gz)
...
error: while evaluating the attribute 'server' of the derivation 'styx-0.7.0' at /nix/store/zvqkq549bwinl6rvhzslqbdx44gbpzc5-latest.tar.gz/derivation.nix:12:3:
attribute 'bin' missing, at /nix/store/zvqkq549bwinl6rvhzslqbdx44gbpzc5-latest.tar.gz/derivation.nix:17:15
```

Similarly unable to preview any themes
```
$ nix-shell -p styx
$ styx preview-theme showcase
error: attribute 'bin' missing, at /nix/store/5v1ybdzyqfv57fvxdb7042gk9bj4xi2a-styx-0.7.1/share/styx-src/derivation.nix:17:15
(use '--show-trace' to show detailed location information)
Please select an available theme, available themes are:
/nix/store/5v1ybdzyqfv57fvxdb7042gk9bj4xi2a-styx-0.7.1/bin/styx: line 421: /revs.csv: No such file or directory
```
```

**Comment by dermetfan** [comment]
Yes, see #41

**Comment by ldeck** [comment]
> Yes, see #41

Ah, great. But that merge request has been there a few months. Any committers able to perform the merge?

**Comment by dermetfan** [comment]
I fear not. The author @ericsagnes has had no activity on GitHub for two years.

You will have to merge it by yourself, like I did in my [fork](https://github.com/dermetfan/styx).

Or use a `shell.nix` that patches it, like this:

```nix
{ pkgs ? import <nixpkgs> {} }:

pkgs.mkShell {
  buildInputs = [
    (pkgs.styx.overrideAttrs (oldAttrs: {
      patches = [
        (pkgs.writeText "41.patch" ''
          diff --git a/derivation.nix b/derivation.nix
          index 34c16be..96e2a9c 100644
          --- a/derivation.nix
          +++ b/derivation.nix
          @@ -17 +17 @@ stdenv.mkDerivation rec {
          -  server = "''${caddy.bin}/bin/caddy";
          +  server = "''${caddy}/bin/caddy";
        '')
      ];
    }))
  ];
}
```

**Comment by ericsagnes** [comment]
should be fixed by #41
